mayan/apps/dynamic_search/queues.py

```diff
@@ -7,6 +7,8 @@
 
 from .literals import TASK_SAVED_RESULTSET_EXPIRED_DELETE_INTERVAL
 
+from .settings import setting_search_disable
+
 queue_search = CeleryQueue(
```

Comment:
This is the queue definition, tried disabling this, throws an error as the tasks (in tasks.py) bind to these queue/task definitions.